src/llmcompressor/transformers/sparsification/compressed_tensors_utils.py

```diff
@@ -77,15 +81,22 @@ def save_pretrained_wrapper(
             if state_dict is None:
                 state_dict = get_state_dict_offloaded_model(model)
 
+            sparsity_stucture = SparsityConfigMetadata.infer_sparsity_structure(model)
```

Comment:
nit: move this into `infer_quantization_format` and copy onto line 99. This way the `infer_quantization_format` has less dependencies

Reply:
Keeping it as is for now!